.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = W503, E501
```

Comment:
We do not have W503 errors.

Regarding E501 - what do you think about fixing it in the code and not ignoring the error here?